jwst/extract_1d/soss_extract/atoca.py

```diff
@@ -1506,6 +1470,9 @@ class ExtractionEngine(_BaseOverlap):
     This version models the pixels of the detector using an oversampled trapezoidal integration.
     """
 
+    # The desired data-type for computations, e.g., 'float32'. 'float64' is recommended.
```

Comment:
did this not already happen in this module? maybe unrelated to the first instance, but consider a refactor to avoid this being specified twice.

Reply:
Yes, there's already an instance of this up above in atoca.py (same module)